PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+- Please include a summary of the change and which issue is fixed. 
+- Please also include relevant motivation and context. 
+- List any dependencies that are required for this change.
+
+## User Story / Task Link
+
+Addresses [AB#XXXXX](url)
+
+## Type of change
+
+Please delete or add options that are or aren't relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
I would add to the end:

`- [ ] I have notified teams of other company projects impacted by this change`

Reply:
```suggestion
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have notified teams of other company projects impacted by this change
```

:)